### ISSUE TITLE
Error while publishing app to Windows store v8jsi.dll is not supported for this application type

### ISSUE THREAD

**Issue by ShivamJoker** [microsoft/react-native-windows#4959]
### Environment

If you are using latest version:
1. `react-native -v`:  "^0.62.0-0",

2. `react-native run-windows --info`:

```
  System:
    OS: Windows 10 10.0.18363
    CPU: (2) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
    Memory: 3.47 GB / 7.98 GB
  Binaries:
    Node: 12.13.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.4 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 6.12.0 - C:\Program Files\nodejs\npm.CMD
  npmPackages:
    react: 16.11.0 => 16.11.0
    react-native: ^0.62.0-0 => 0.62.2
    react-native-windows: ^0.62.0-0 => 0.62.0-preview.6
  Installed UWP SDKs:
    10.0.17763.0
    10.0.18362.0
    10.0.19041.0
```
3. `reg query "HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock"`

```
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock
    AllowDevelopmentWithoutDevLicense    REG_DWORD    0x1
    AllowAllTrustedApps    REG_DWORD    0x1
```

Then, specify:

- Target Platform Version(s):

- Target Device(s):
Desktop

- Visual Studio Version: 2019 

- Build Configuration: ReleaseBundle

### Steps to Reproduce

This is a simple Rock Paper Scissors Game made in react native no extra modules

1. Clone the repo https://github.com/ShivamJoker/RN-Rock-Paper-Scissors
2.
 ```
cd RN-Rock-Paper-Scissors
npx react-native-windows-init --overwrite
```
3. Project > Publish > Create App Package > For Windows Store

After I got the `RockPaperScissors_1.0.6.0_x86_x64.appxbundle`
I uploaded it to newly created microsoft dev account
after going through all the process I got a certification error for Supported Api test

## Error which I got

```
API ?makeV8Runtime@v8runtime@@YA?AV?$unique_ptr@VRuntime@jsi@facebook@@U?$default_delete@VRuntime@jsi@facebook@@@std@@@std@@$$QEAUV8RuntimeArgs@1@@Z in v8jsi.dll is not supported for this application type. Microsoft.ReactNative.dll calls this API.
```

The same error I got when I ran the Windows App Certification Kit

I am trying out all these for the first time so please help me what should I do ? I am stuck with this from 3 days

**Comment by NickGerleman** [comment]
#4475 was supposed to fix publish issues. @asklar had some more comprehensive changes around V8 inclusion. We will likely need to backport these to 0.62.

FYI @tudorms

**Comment by ShivamJoker** [comment]
So is there any way for me to fix it for now ?

**Comment by NickGerleman** [comment]
As a quick workaround, would you be able to try editing your `node_modules/react-native-windows/PropertySheets/React.Cpp.props` and replace "true" in this line with "false"?

```
<USE_V8 Condition="'$(USE_V8)' == '' AND '$(Platform)' != 'ARM' AND '$(Platform)' != 'ARM64'">true</USE_V8>
```

We will try to push a fix for this as soon as possible so that you won't have to do this.

**Comment by NickGerleman** [comment]
This will also likely affect 0.60, 0.61. @tudorms can you confirm?

**Comment by tudorms** [comment]
Yes, it looks like we'll need to include  https://github.com/microsoft/react-native-windows/pull/4820 and https://github.com/microsoft/react-native-windows/pull/4840 in 0.62 (the changes were made 2 weeks ago).

If 0.62 is already out we probably don't need to worry about 0.60 and 0.61.

**Comment by NickGerleman** [comment]
We're still backporting some fixes to 0.60 and 0.61, so we'll need to have a discussion there. @tudorms  would you be willing to start a PR to recreate those two changes in 0.62 in the meantime?

**Comment by chrisglein** [comment]
What I'm hearing is that this is fixed, but isn't in `0.62.0-preview.6`. We're up to `.8` now and hope to be releasing soon. Please try again with latest (or when we release) and let us know if you're unblocked.

**Comment by asklar** [comment]
> So is there any way for me to fix it for now ?

@ShivamJoker can you try using a newer 62? e.g. anything newer than preview 8, e.g. https://www.npmjs.com/package/react-native-windows/v/0.62.0-preview.11
